static/app/components/feedback/useFetchFeedbackData.tsx

```diff
@@ -40,7 +40,9 @@ export default function useFetchFeedbackData({feedbackId}: Props) {
   const {markAsRead} = useMutateFeedback({
     feedbackIds: [feedbackId],
     organization,
-    projectIds: issueData ? [issueData?.project.id] : [],
+    // Typescript is a lie. `project` might be missing if feedback didn't go
+    // through the new ingest pipeline. This can happen in new self-hosted upgrades.
```

Comment:
I'm confused when exactly the project gets dropped, and what's this have to do with self-hosted?

Reply:
What's happening on self-hosted is that people are not having `issueData.project` set for all their feedback instances. 
I hope & believe that it's because old feedbacks (not processed by the new pipelines) are still in the database and don't have everything wired up because they were ingested using the old pipeline. There's no migration to fix things up.